.gitattributes

```diff
@@ -60,3 +61,6 @@ CMakeLists.txt text whitespace=tab-in-indent,trail,space
 
 *.xml -text
 *.yml -text
+*.patch -text
```

Comment:
This and the change above are needed to avoid line-ending changes that cause patch application & package building to fail.